by-language/php-amphp/README.rst

```diff
@@ -0,0 +1,57 @@
+.. highlight: console
+
+###########################################################
+Connect to CrateDB and CrateDB Cloud using AMPHP/PostgreSQL
+###########################################################
+
+
+*****
+About
+*****
+
+This is an example program demonstrating CrateDB's `PostgreSQL wire protocol`_
+compatibility by exercising a few basic SQL statements using `AMPHP PostgreSQL
+driver`_, based on the `AMPHP`_ framework (see also `AMPHP on GitHub`_). It is
+an asynchronous PostgreSQL client for PHP.
+
+For further information, please also visit the `CrateDB clients and tools`_
+page. Because CrateDB only supports (implicitly created) `table schemas`_
+instead of databases, please also read `PostgreSQL documentation about
+schemas`_ carefully.
+
+Also, please note that support for this driver is experimental. The `CrateDB
+PDO driver`_ is the official CrateDB PHP client supported by Crate.io.
```

Comment:
Any reason not to use a `.. note::` section here?

```suggestion
.. note::

    The support for this driver is experimental. The `CrateDB PDO driver`_ 
    is the official CrateDB PHP client supported by Crate.io.
```

Reply:
Thank you!